tests/dummy/app/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Ember Page Title</title>
+    <title data-test-ignore-for-fastboot-tests>Ember Page Title</title>
```

Comment:
Since we are keeping original title element for non-fastboot vs fastboot tests - we need to filter it out for assertions.